lib/index.js

```diff
@@ -65,6 +65,7 @@ Elevio.prototype.identify = function(identify) {
   if (email) user.email = email;
   if (name) user.name = name;
   if (plan) user.plan = [plan];
+  if (plan) user.groups = [plan];
```

Comment:
cheers @hankim813 this is simply to allow our new version of the embeddable access to the plan information. In future the user.plan will be deprecated at our (elevio's) end, this is just a transition PR.

In future a similar PR will be made that will remove the user.plan alltogether, and introduce the ability for segment users to explicitly set the user.groups value rather than using the plan as it currently does, for more clarity and control

Most importantly: there will be no flow on of unexpected behaviour to any customers.